manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/tooling/Document.scala

```diff
@@ -202,6 +205,24 @@ case class Query(queryText: String, assertions: QueryAssertions, myInitQueries:
     content.runnableContent(initQueries ++ myInitQueries :+ queryText)
 }
 
+case class ConsoleData(globalInitQueries: Seq[String], localInitQueries: Seq[String], query: String) extends Content with NoQueries {
+  override def asciiDoc(level: Int): String = {
+    val globalInitQueryRows = globalInitQueries.mkString(NewLine)
+    val localInitQueryRows = localInitQueries.mkString(NewLine)
+    s""".Try this query live
+       |[console]
+       |----
+       |$globalInitQueryRows
+       |
```

Comment:
Perfect. I've given you write access to my fork now so that you won't have to open a new pull request next time :)